src/main/resources/assets/opencomputers/doc/en_US/general/lua.md

```diff
@@ -0,0 +1,12 @@
+# Lua 
+
+The Lua [reference manual](www.lua.org/manual/5.2/manual.html) is a good place to get started with the basics of Lua and becoming familiar with the basic syntax and standard libraries. OpenComputers has very similar standard libraries, and omits certain libraries entirely (such as the debug library). 
```

Comment:
External links have to be prefixed with `http://` or `https://` to be recognized.

Minor correction: debug library is missing 99%, but traceback is available.